tests/utils/test_async_logging_queue.py

```diff
@@ -43,6 +44,46 @@ def consume_queue_data(self, run_id, metrics, tags, params):
 
 
 def test_single_thread_publish_consume_queue():
+    from mlflow.environment_variables import MLFLOW_ASYNC_LOGGING_WAITING_TIME
+
+    MLFLOW_ASYNC_LOGGING_WAITING_TIME.set("3")
```

Comment:
```suggestion
    monkeypatch.setenv(MLFLOW_ASYNC_LOGGING_WAITING_TIME, "3")
```

nit: so we don't need to clear it manually